common-utils/src/common/deployment/__init__.py

```diff
@@ -131,10 +117,7 @@ def get_pyproject_metadata() -> PyProjectMetadata:
     )
 
 
-PYPROJECT_METADATA = get_pyproject_metadata()
```

Comment:
removing the pyproject metadata get on import of this module.  Should not need to do this until running deploy-cli command directly.